### ISSUE TITLE
[Screen Reader-Social Embedded-Social Activities]: TalkBack does not read the reply and comment text for the post.

### ISSUE THREAD

**Issue by billycamp** [Microsoft/EmbeddedSocial-Android-SDK#21]
Priority: High

Repro Steps
1. Launch the One Bus Away App.
2. Navigate to ""Sign In"" link under Hamburger link and press ENTER.
3. Sign in screen showing all currently supported identity service providers will appear.
4. Use any service provider for log in.
5. Select any location bubble and see the given arrivals.
6. Navigate to ""Join Discussion"" button and press ENTER.
7. Create any profile and comment on that profile post.
8. Navigate to ""reply"" button and press ENTER to reply on that comment.
9. Verify the speech of talkback for the Reply text.

Note:
1. Talkback focus is moving to the entire comment section first and read everything, however when talkback focus moves to separate text and controls, talkback does not read anything. Either focus should not move to the separate elements and if focus move to different elements, then talkback should read every elements.

Expected
TalkBack should read the Reply and Comment text for the post. What ever text is there under Reply section, talkback should read that text. 

Actual
TalkBack does not read the Reply and Comment text for the post.

MAS Rule: MAS4.2.7

Impact
If talkback does not read the reply text, then screen reader users will not be able to read the reply for any post.

Test Environment
OS: Android
Device Model No. : Nexus 5
Android Version : 6.0.1
App: Social Embedded
App Version : v2.2.3 (84)
Tool : Talk Back
